c/common/src/codingstandards/c/Expr.qll

```diff
@@ -3,8 +3,19 @@ import cpp
 /* A full expression as defined in ISO/IEC 9899:2011 6.8 point 4 and Annex C point 1 item 5. */
 class FullExpr extends Expr {
   FullExpr() {
-    not this.getParent() instanceof Expr and
```

Comment:
you probably already saw this, but this change effected the lib test for this, so that will require a relook and reaccept